cmd/opdoc/opdoc.go

```diff
@@ -91,7 +91,7 @@ func namedStackTypesMarkdown(out io.Writer, stackTypes []namedType) {
 	for _, st := range stackTypes {
 		fmt.Fprintf(out, "| %s | %s | %s |\n", st.Name, st.boundString(), st.AVMType)
 	}
-	out.Write([]byte("\n"))
+	fmt.Fprintf(out, "\n")
```

Comment:
what about the others?

Reply:
Other what?

I only changed this because the linter allows ignoring error returns from printf, but not write.

Reply:
oh.